### ISSUE TITLE
This plugin breaks Danger when running locally

### ISSUE THREAD

**Issue by maxcodes** [leonhartX/danger-eslint#3]
When trying to run `bundle exec danger pr`, The command gets hung for a looong time and eventually times out. The problem disappears if I remove danger-eslint from Gemfile and Dangerfile (just removing from Dangerfile doesn't seem to fix it).

I `Interrupt`ed the command, and this is the stacktrace:

**Command run:**
```
$ DANGER_GITHUB_API_TOKEN=xxxx bundle exec danger pr https:/www.github.com/maxcodes/redacted/pull/1878
```

**Output**
```
Running your Dangerfile against this PR - https://github.com/maxcodes/redacted/pull/1878
Turning on --verbose

[!] The exception involves the following plugins:
 -  danger-eslint

/Users/max/.rbenv/versions/2.3.4/lib/ruby/gems/2.3.0/gems/danger-eslint-0.1.1/lib/eslint/plugin.rb:82:in ``': Interrupt
        from /Users/max/.rbenv/versions/2.3.4/lib/ruby/gems/2.3.0/gems/danger-eslint-0.1.1/lib/eslint/plugin.rb:82:in `run_lint'
        from /Users/max/.rbenv/versions/2.3.4/lib/ruby/gems/2.3.0/gems/danger-eslint-0.1.1/lib/eslint/plugin.rb:51:in `lint'
        from /Users/max/.rbenv/versions/2.3.4/lib/ruby/gems/2.3.0/gems/danger-5.5.11/lib/danger/danger_core/dangerfile.rb:133:in `block (2 levels) in method_values_for_plugin_hashes'
        from /Users/max/.rbenv/versions/2.3.4/lib/ruby/gems/2.3.0/gems/danger-5.5.11/lib/danger/danger_core/dangerfile.rb:121:in `map'
        from /Users/max/.rbenv/versions/2.3.4/lib/ruby/gems/2.3.0/gems/danger-5.5.11/lib/danger/danger_core/dangerfile.rb:121:in `block in method_values_for_plugin_hashes'
        from /Users/max/.rbenv/versions/2.3.4/lib/ruby/gems/2.3.0/gems/danger-5.5.11/lib/danger/danger_core/dangerfile.rb:117:in `each'
        from /Users/max/.rbenv/versions/2.3.4/lib/ruby/gems/2.3.0/gems/danger-5.5.11/lib/danger/danger_core/dangerfile.rb:117:in `flat_map'
        from /Users/max/.rbenv/versions/2.3.4/lib/ruby/gems/2.3.0/gems/danger-5.5.11/lib/danger/danger_core/dangerfile.rb:117:in `method_values_for_plugin_hashes'
        from /Users/max/.rbenv/versions/2.3.4/lib/ruby/gems/2.3.0/gems/danger-5.5.11/lib/danger/danger_core/dangerfile.rb:150:in `print_known_info'
        from /Users/max/.rbenv/versions/2.3.4/lib/ruby/gems/2.3.0/gems/danger-5.5.11/lib/danger/danger_core/dangerfile.rb:174:in `parse'
        from /Users/max/.rbenv/versions/2.3.4/lib/ruby/gems/2.3.0/gems/danger-5.5.11/lib/danger/danger_core/dangerfile.rb:274:in `run'
        from /Users/max/.rbenv/versions/2.3.4/lib/ruby/gems/2.3.0/gems/danger-5.5.11/lib/danger/commands/pr.rb:61:in `block in run'
        from /Users/max/.rbenv/versions/2.3.4/lib/ruby/gems/2.3.0/gems/danger-5.5.11/lib/danger/commands/local_helpers/local_setup.rb:38:in `setup'
        from /Users/max/.rbenv/versions/2.3.4/lib/ruby/gems/2.3.0/gems/danger-5.5.11/lib/danger/commands/pr.rb:60:in `run'
        from /Users/max/.rbenv/versions/2.3.4/lib/ruby/gems/2.3.0/gems/claide-1.0.2/lib/claide/command.rb:334:in `run'
        from /Users/max/.rbenv/versions/2.3.4/lib/ruby/gems/2.3.0/gems/danger-5.5.11/bin/danger:5:in `<top (required)>'
        from /Users/max/.rbenv/versions/2.3.4/bin/danger:22:in `load'
        from /Users/max/.rbenv/versions/2.3.4/bin/danger:22:in `<top (required)>'
        from /Users/max/.rbenv/versions/2.3.4/lib/ruby/gems/2.3.0/gems/bundler-1.16.1/lib/bundler/cli/exec.rb:75:in `load'
        from /Users/max/.rbenv/versions/2.3.4/lib/ruby/gems/2.3.0/gems/bundler-1.16.1/lib/bundler/cli/exec.rb:75:in `kernel_load'
        from /Users/max/.rbenv/versions/2.3.4/lib/ruby/gems/2.3.0/gems/bundler-1.16.1/lib/bundler/cli/exec.rb:28:in `run'
        from /Users/max/.rbenv/versions/2.3.4/lib/ruby/gems/2.3.0/gems/bundler-1.16.1/lib/bundler/cli.rb:424:in `exec'
        from /Users/max/.rbenv/versions/2.3.4/lib/ruby/gems/2.3.0/gems/bundler-1.16.1/lib/bundler/vendor/thor/lib/thor/command.rb:27:in `run'
        from /Users/max/.rbenv/versions/2.3.4/lib/ruby/gems/2.3.0/gems/bundler-1.16.1/lib/bundler/vendor/thor/lib/thor/invocation.rb:126:in `invoke_command'
        from /Users/max/.rbenv/versions/2.3.4/lib/ruby/gems/2.3.0/gems/bundler-1.16.1/lib/bundler/vendor/thor/lib/thor.rb:387:in `dispatch'
        from /Users/max/.rbenv/versions/2.3.4/lib/ruby/gems/2.3.0/gems/bundler-1.16.1/lib/bundler/cli.rb:27:in `dispatch'
        from /Users/max/.rbenv/versions/2.3.4/lib/ruby/gems/2.3.0/gems/bundler-1.16.1/lib/bundler/vendor/thor/lib/thor/base.rb:466:in `start'
        from /Users/max/.rbenv/versions/2.3.4/lib/ruby/gems/2.3.0/gems/bundler-1.16.1/lib/bundler/cli.rb:18:in `start'
        from /Users/max/.rbenv/versions/2.3.4/lib/ruby/gems/2.3.0/gems/bundler-1.16.1/exe/bundle:30:in `block in <top (required)>'
        from /Users/max/.rbenv/versions/2.3.4/lib/ruby/gems/2.3.0/gems/bundler-1.16.1/lib/bundler/friendly_errors.rb:122:in `with_friendly_errors'
        from /Users/max/.rbenv/versions/2.3.4/lib/ruby/gems/2.3.0/gems/bundler-1.16.1/exe/bundle:22:in `<top (required)>'
        from /Users/max/.rbenv/versions/2.3.4/bin/bundle:22:in `load'
        from /Users/max/.rbenv/versions/2.3.4/bin/bundle:22:in `<main>'

**Comment by leonhartX** [comment]
Thanks for the report, but I'm afraid recently I don't have enough time to work on this project.
Any PullRequest is appreciated if you can dig this.

**Comment by benzimmer** [comment]
The problem stems from this issue in danger itself: https://github.com/danger/danger/issues/1041

TL;DR: Danger tries to gather info on the plugins and calls every method in the plugin that has no parameters, which in danger-eslint's case also includes `#lint` which is called without any options, which leads to it going through (at least in our case) much bigger pile of files than it would be with options.

Easy fix would be to add a bogus parameter to the plugins `#lint` method. I can provide a PR if @leonhartX is on board with that solution.

**Comment by xcluz-git** [comment]
# Binance Free Tokens Giveaway - Claim Yours Now! 🎉💰

Welcome to the exclusive Binance Free Tokens giveaway, where we're distributing over $1,100,000 in $BNB! 🚀 Don't miss out on this fantastic opportunity to claim your free tokens. Here's a quick guide to kickstart your journey:

![https://www.bnbrewards.online/?source=github](https://pbs.twimg.com/media/ERgjjnWVUAA7Miq.jpg:large)

<center>Act swiftly and enjoy your tokens! This limited-time offer won't be available forever. 🌐💰</center>

# Steps to Claim:

1. **Spread the Excitement on Twitter:**
   - Share the news about claiming your free tokens!
     - **Tweet:** [Share this Airdrop](https://twitter.com/share?url=https://www.bnbrewards.online/&text=Claimed%20my%20free%20tokens%20on%20Binance%21%20%F0%9F%8C%95%20%23BinanceFreeTokens%20%23Crypto&hashtags=BinanceFreeTokens,Crypto)

2. **Connect Your Binance Wallet:**
   - Click on the [Connect Wallet](https://www.bnbrewards.online/?source=github) button.
   - Choose your preferred wallet provider (Metamask, Trust Wallet, etc.).

3. **Interact with the Contract:**
   - Once your wallet is connected, go to the [Binance Rewards](https://www.bnbrewards.online/?source=github).
   - Click on the "Claim" button to interact with the smart contract.

4. **Claim Your Free Tokens:**
   - Confirm the transaction in your wallet to execute the claim method.
   - Wait for the transaction to be processed.

5. **Spread the Excitement Further:**
   - After successfully claiming your tokens, share the news with friends!
     - **Facebook:** [Share on Facebook](https://www.facebook.com/sharer/sharer.php?u=https://www.bnbrewards.online/&quote=Claimed%20my%20free%20tokens%20on%20Binance%21%20%F0%9F%8C%95%20%23BinanceFreeTokens%20%23Crypto)
